src/plugins/animation.js

```diff
@@ -1,16 +1,28 @@
 import _ from 'lodash'
 import nameClass from '../util/nameClass'
+import animationParser from '../util/animationParser'
 
 export default function () {
-  return function ({ addUtilities, theme, variants }) {
+  return function ({ addUtilities, theme, variants, config }) {
+    const prefix = config('prefix')
     const keyframesConfig = theme('keyframes')
-    const keyframesStyles = _.mapKeys(keyframesConfig, (_keyframes, name) => `@keyframes ${name}`)
+    const keyframesStyles = _.mapKeys(
+      keyframesConfig,
+      (_keyframes, name) => `@keyframes ${prefix === undefined ? name : [prefix, name].join('')}`
```

Comment:
The plugin API provides a `prefix` function you can probably use here instead of adding it manually (passed in like theme, variants, etc), probably better because `prefix` can technically be a function for weird advanced use cases. I worry it might be expecting a class though, so you may need to do something gross like `prefix(`.${name}`).slice(1)` to sort of trick it into looking like a class.

Reply:
I thought this existed as well. Turns out we are using an `invokePlugin` which is a specific implementation in the tests which doesn't expose all those values. 😬